Core/src/org/sleuthkit/autopsy/logicalimager/configuration/ConfigVisualPanel1.java

```diff
@@ -257,7 +281,8 @@ private void refreshDriveList() {
             String description = FileSystemView.getFileSystemView().getSystemTypeDescription(root);
             long spaceInBytes = root.getTotalSpace();
             String sizeWithUnit = DriveListUtils.humanReadableByteCount(spaceInBytes, false);
-            listData.add(root + " (" + description + ") (" + sizeWithUnit + ")");
+            String fileSystem = getFileSystemName(root.toString());
+            listData.add(root + " (" + description + ") (" + sizeWithUnit + ") - " + Bundle.ConfigVisualPanel1_fileSystem() + ": " + fileSystem);
```

Comment:
This might be cleaner if you put the whole thing into the Bundle message and used variables where needed